src/metabase/models/native_query_snippet.clj

```diff
@@ -35,10 +28,11 @@
   i/IObjectPermissions
   (merge
    i/IObjectPermissionsDefaults
-   {:perms-objects-set perms-objects-set
-    :can-read?         (partial i/current-user-has-full-permissions? :write)
-    :can-write?        (partial i/current-user-has-full-permissions? :write)
-    :can-create?       (partial i/current-user-has-full-permissions? :write)}))
+   {;; In Metabase CE, everyone can read all NativeQuerySnippets; only admins can create or edit them. In EE
+    ;; permissions are dictated by a 'folder' system similar to Collections (not yet implemented).
```

Comment:
I thought that in CE anyone could create/edit snippets. I like this behavior better (feels more consistent with metrics/segments) but want to confirm what the plan is. cc @mazameli

Reply:
Yes, in CE, any user can create, edit, archive, and unarchive snippets from the native query editor screen.

Reply:
@camsaul should I just change these `i/superuser?` to `(constantly true)` or would you want to do a bigger change?

Reply:
@paulrosenzweig I can tackle that really quick